mappings/servers/akumamc/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "akumamc",
+    "name": "AkumaMC",
+    "description": "AkumaMC is the best Minecraft OP Prison & Skyblock Server out there. Your experience is our #1 priority.",
+    "addresses": [
+        "akumamc.net",
+        "akumamc.com"
+    ],
+    "primaryAddress": "akumamc.net",
+    "minecraftVersions": [
+        "1.8.9",
+        "1.21.5"
+    ],
+    "primaryMinecraftVersion": "1.21.5",
```

Comment:
The primary version needs to be a version CheatBreaker supports, so set it to 1.8.9